drizzlepac/buildwcs.py

```diff
@@ -381,10 +381,10 @@ def generate_headerlet(outwcs,template,wcsname,outname=None):
         if outname.find('_hdr.fits') < 0:
             outname += '_hdr.fits'
         if os.path.exists(outname):
-            print('Overwrite existing file "%s"'%outname)
+            log.debug('Overwrite existing file "%s"'%outname)
             os.remove(outname)
         hdrlet.writeto(outname)
-        print('Wrote out headerlet :',outname)
+        log.debug('Wrote out headerlet :',outname)
```

Comment:
This message will also throw an error.  Something like this would work:

```suggestion
        log.debug('Wrote out headerlet : %s',outname)
```